assets/src/data/model/datetime/index.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * External imports
+ */
+import moment from 'moment';
+import { isUndefined } from 'lodash';
+import PropTypes from 'prop-types';
+
+import {
+	getQueryString as baseGetQueryString,
+} from '../base';
+
+export const nowDateAndTime = moment();
+
+/**
+ * Described attributes for this model
+ * @type {{attributes: *}}
+ */
+export const queryDataTypes = {
+	queryData: PropTypes.shape( {
+		limit: PropTypes.number,
+		orderBy: PropTypes.oneOf( [
+			'DTT_name',
+			'DTT_ID',
+			'start_date',
+			'end_date',
+		] ),
+		order: PropTypes.oneOf( [ 'asc', 'desc' ] ),
+		showExpired: PropTypes.bool,
+		month: PropTypes.month,
+	} ),
+};
+
+/**
+ * Default attributes for this model
+ * @type {{attributes: {limit: number, orderBy: string, order: string,
+ *   showExpired: boolean}}}
```

Comment:
maybe this could be formatted better?

Reply:
ya will do.  This was done automatically by PhpStorm but doesn't look like there is a way to fix the auto-formatting for this so manual fixes will be necessary.